src/cordova/apple/OutlineAppleLib/Sources/OutlineAppKitBridge/StatusItemController.swift

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if canImport(AppKit)
+import AppKit
+import CocoaLumberjackSwift
+import OutlineShared
+
+var StatusItem = NSStatusItem()
+
+class StatusItemController: NSObject {
+    let connectionStatusMenuItem = NSMenuItem(title: MenuTitle.statusDisconnected,
+                                              action: nil,
+                                              keyEquivalent: "")
+
+    private enum AppIconImage {
+        static let statusConnected = NSImage(named: NSImage.Name("StatusBarButtonImageConnected"))!
```

Comment:
This is a hidden dependency on the app code, which is a circular dependency. Can you move the Assets.xcassets into `OutlineAppleLib`? That will also reduce code in the hard-to-maintain xcode directory.

It's ok if it's in a separate PR, but we should at least add a TODO here.